spec/factories/file_delivery_methods.rb

```diff
@@ -0,0 +1,34 @@
+FactoryBot.define do
+
```

Comment:
If I do this I'd drop the alias and define another specific-delivery class for `upload_now`.  I think this works OK and, if any test actually _cares_ about default vs non-default, it should be using the specific-delivery factories anyway.

Reply:
Hm.  I still think you should set the default so that the test never accidentally gets into the position (state) where the value is not set.  (= not only an indeterminate state, but also might provide bad results for someone writing a test that doesn't realize the state hasn't been set.)
OK with going your way, though.